SwiftPkgsExample/JustPassMeExample/Auth/AuthenticatedView.swift

```diff
@@ -78,7 +79,17 @@ struct AuthenticatedView<Content, Unauthenticated>: View where Content: View, Un
         }
         if viewModel.emailLinkStatus == .none {
           Text("You need to be logged in to use this app.")
-          loginButton("Log in")
+          HStack{
+            loginButton("Log in")
+            if viewModel.hasPasskeys {
+              SignInPassKeysButton(action: {
+                Task {
+                  await viewModel.loginPasskeys(window: sceneDelegate.window!, autofill: false)
```

Comment:
We can use `.init()` instead, and it's advisable to avoid force unwrapping to prevent potential crashes.